index.js

```diff
@@ -28,6 +28,9 @@ module.exports = class Trailpack {
     if (!pack.config) {
       pack.config = { }
     }
+    if (!pack.config.trailpack) {
```

Comment:
@jaumard  If you dont pass a configuration for the trailpack, create a empty object